packages/admin-panel/src/editor/EditModal.js

```diff
@@ -48,6 +48,7 @@ export const EditModalComponent = ({
   cancelButtonText,
   saveButtonText,
   extraDialogProps,
+  endpoint,
```

Comment:
Are these your changes?

Reply:
yes, they were before creating the resubmit modal. I'll remove that prop.